robot_name: 'cybernetic_pirate'
# This goes to making up part of the platform_info triple.
robot_type: 'turtlebot'

# Auto start a rapp, e.g. 'rocon_apps/chirp'.
# This doesn't do remappings like the start_app service (yet).
auto_start_rapp: ''

# Whitelist/blacklists for who can take control of the app manager
# These are currently lists of remote gateway names...
remote_controller_whitelist: []
remote_controller_blacklist: []

# Useful for local machine tests only such as chatter_concert.
# It is an easy way of preventing your app managers from being stolen by
# other auto-inviting concerts.
local_remote_controllers_only: false

# Semi colon separated string. This is very non-portable, use launchers where you can use $(find..) instead
# rapp_lists: '/home/jihoonl/ros/groovy/turtlebot/turtlebot_apps/turtlebot_core_apps/turtlebot.rapps'

# Uncomment and modify below to explicitly set the preferred
# implementation for an ancestor rapp specification:
#  - rocon_apps/chirp: 'rocon_apps/moo'
# and when an ancestor is an implementation...
#  - rocon_apps/talker: 'rocon_apps/talker'
